### ISSUE TITLE
Please consider removing custom `install` script

### ISSUE THREAD

**Issue by lensbart** [tsparticles/tsparticles#3502]
### What do you need?

Hello,

First of all, thank you maintaining this package. It’s great!

`tsparticles` and `react-tsparticles` use a custom `install` script to display some console messages and check for the presence of a deprecated `react-particles-js`. Having a custom install script is inconvenient:

* The default behaviour of [Yarn PnP](https://next.yarnpkg.com/features/pnp/) is to not compress these packages, and to include them wholesale in any consuming repository (11MB on disk). This is checked in to version control and included in every deploy.
* It’s possible to override this behaviour by specifying the following in `package.json`:
  ```
  "dependenciesMeta": {
    "react-tsparticles": {
      "built": false
    },
    "tsparticles": {
      "built": false
    }
  },
  ```
  but this leads to the following warnings to be displayed by Yarn:
  ```
  ➤ YN0007: │ tsparticles@npm:1.42.1 must be built because it never has been before or the last one failed
  ➤ YN0007: │ react-tsparticles@npm:1.42.1 [e18ad] must be built because it never has been before or the last one failed  ```
* It’s possible to disable these warning by adding the following to `.yarnrc.yml`:
  ```
  logFilters:
    - code: YN0005
      level: discard
  ```
  but this would disable all such warnings, i.e. for other packages as well.

In other words, no convenient option exists to have minimum-disk-size experience (& distraction-free) experience.

(Also, `react-particles-js` has been deprecated for a while, so most users have either migrated or decided to ignore the warning.)

A better explanation of why to avoid postinstall can be found here: https://yarnpkg.com/advanced/lifecycle-scripts/#a-note-about-postinstall

Thank you!

### Version

1.42.1

### Which library are you using?

Vanilla (tsparticles), React.js (react-tsparticles, react-particles-js)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
